code/notificationbuilder/src/main/res/layout/push_template_expanded_api23.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/basic_expanded_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:clipToPadding="false"
+    android:clipChildren="false"
+    android:orientation="vertical"
+    android:theme="@style/DayNightTheme">
+
+    <ImageView
+        android:id="@+id/large_icon"
+        android:layout_height="@dimen/large_icon_height"
+        android:layout_width="@dimen/large_icon_width"
+        android:layout_alignParentRight="true" />
+
+    <TextView
+        android:id="@+id/notification_title"
+        style="@style/Title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="start"
+        android:textAlignment="viewStart"
+        android:layout_toLeftOf="@+id/large_icon"/>
+
+    <TextView
+        android:id="@+id/notification_body_expanded"
+        style="@style/Body"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/notification_title"
+        android:layout_gravity="start"
+        android:textAlignment="viewStart"
+        android:layout_toLeftOf="@+id/large_icon"/>
+
+    <ImageView
+        android:id="@+id/expanded_template_image"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/basic_expanded_image_height_api23"
```

Comment:
Wondering if this can be solved with a single layout for all API levels by using weights. If we set `layout_height=0` and `layout_weight=1` so it takes up remaining space would that work and still leave space for buttons?

Reply:
Discussed offline, this is not working

Reply:
discussed and resolved this via slack